certification/README.md

```diff
@@ -0,0 +1,2 @@
+# certification
+Software Carpentry certifications
```

Comment:
This is a stripped down version of the script, modified to fit with the current workflow. However, I'll add the link anyway. It deserves credits!